src/plugins/plugin_integration/opensearch_dashboards.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "pluginIntegration",
+  "version": "opensearchDashboards",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": ["data", "savedObjects", "opensearchDashboardsUtils"]
```

Comment:
Can we try a `yarn osd bootstrap` & `yarn start`? savedObjects seems not been used at the moment, build should fail for redundant requiredPlugins.

Reply:
I had & just re-ran, haven't seen any errors. `savedObjects` will be needed in the next PRs, is it ok for me to leave in for now?

Reply:
If build has no issues then I am good.